json-rpc/types/src/views.rs

```diff
@@ -65,6 +65,7 @@ pub enum AccountRoleView {
         num_children: u64,
         compliance_key_rotation_events_key: EventKey,
         base_url_rotation_events_key: EventKey,
+        diem_id_domains: DiemIdDomains,
```

Comment:
please also add the data structure to json-rpc/types/src/proto/jsonrpc.proto
and add test to confirm the rendered result can be loaded by the type generated from proto file.